src/pages/docs/messages/annotations.mdx

```diff
@@ -0,0 +1,418 @@
+</Aside>
+
+When publishing a delete annotation, specify the [annotation type](#annotation-types) on the `type` field of the annotation object.
+
+You can also optionally specify a `name` for the annotation which is used by certain [summarization methods](#annotation-summaries) that aggregate by `name`.
+
+<Code>
+```javascript
+// Delete a 'delivered' annotation
+await channel.annotations.delete(message.serial, {
+  type: 'receipts:flag.v1',
+  name: 'delivered'
+});
+```
+
+```nodejs
+// Delete a 'delivered' annotation
+await channel.annotations.delete(message.serial, {
+  type: 'receipts:flag.v1',
+  name: 'delivered'
+});
+```
+</Code>
+
+
+## Subscribing to annotation updates <a id="subscribe" />
+
+There are two ways to subscribe to annotation updates:
+
+1. **Annotation summaries** (recommended) - Receive aggregated summaries of all annotations for a message
+2. **Individual annotations** - Receive individual events when annotations are published or deleted
+
+### Subscribing to annotation summaries <a id="subscribe-annotation-summaries" />
+
+The recommended way to receive annotation updates is through annotation summaries. These events provide a summary of the complete, current state of all annotations for a message whenever annotations are published or deleted.
+
+Annotation summaries are delivered to subscribers as messages with an `action` of `message.summary`. These messages have a `summary` field which provides a summary of all the annotations for the message identified by the `serial` field on the summary message.
+
+The value of the `summary` field is an object where the keys are the [annotation types](#annotation-types). The exact structure of the value of each key depends on the [summarization method](#annotation-summaries) specified in the annotation type.
+
+<Code>
+```javascript
+await channel.subscribe((message) => {
+  if (message.action === 'message.summary') {
+    console.log(message.summary);
+  }
+});
+```
+
+```nodejs
+await channel.subscribe((message) => {
+  if (message.action === 'message.summary') {
+    console.log(message.summary);
+  }
+});
+```
+</Code>
+
+<Aside data-type="info">
+When using message annotations, normal messages delivered to the [`subscribe()`](/docs/pub-sub#subscribe) listener will have an `action` of `message.create`.
+</Aside>
+
+For more details on summary structure and usage, see [annotation summaries](#annotation-summaries).
+
+### Subscribing to individual annotations <a id="subscribe-individual-annotations" />
+
+For more fine-grained visibility, you can also subscribe directly to individual annotation events using the `annotations.subscribe()` method on a channel.
+
+Annotations delivered to the `annotations.subscribe()` listener will have an `action` of `annotation.create` or `annotation.delete`.
+
+<Code>
+```javascript
+await channel.annotations.subscribe((annotation) => {  
```

Comment:
The code example should include getting the channel with channeloptions that specify the annotation_subscribe mode

Reply:
Fixed in https://github.com/ably/docs/pull/2642/commits/829b3a4044ec8199cd1e409aeb7599602dbd6e0c